### DISCUSSION TITLE
话说各位有没有遇到过打开Shaft或PixEz这类P站app，在登录时，整个软件卡住的情况？

### DISCUSSION THREAD

**Discussion by iPixelOldC** [LSPosed/MagiskOnWSALocal#488] [Q&A]
话说各位有没有遇到过打开Shaft或PixEz这类P站app，在点击软件login按钮进行登录时，整个软件卡住的情况？(Pixiv的登录网页无法显示）我在用微软商店原版的时候登录就很顺利，没有这个问题。（所以就在这里提问了，因为我用原版确实没这问题）
网络肯定是没问题的，本机上Pixeval这个Win下P站软件是能正常用Webview2登录的。不过搜cfw的log，里面确实没有pixiv.net的解析和连接记录，所以或许也可能就是网络配置问题
有点头疼，求各位大佬帮帮忙，分享分享看法。谢谢:D

更新：突然发现点击PixEz右上角那个弹出按钮，可以弹到Firefox上登录，然后就登录成功了，看着还真可能是系统默认的WebView的问题。

我用的是这个版本`WSA_2302.40000.9.0_x64_Release-Nightly-with-magisk-26.1(26100)-stable-MindTheGapps-13.0-RemovedAmazon.7z`,系统是`Win11 22H2 22621.1485`,App的版本都是其官方仓库的最新版本，同时在Play商店下载的PixEz也会出现这个情况。在Play商店也更新下载了最新的Android Webview，为了防止是浏览器的问题，我Firefox也下载了，并设置为了默认浏览器。
卡住的软件截图如下：
<img width="343" alt="2023-04-12_171209" src="https://user-images.githubusercontent.com/13417924/231412091-5525c8fc-18f3-4375-9a94-7c1eba64e0a3.png">

<img width="368" alt="2023-04-12_171136" src="https://user-images.githubusercontent.com/13417924/231412678-0f8e13ab-e758-4c6e-ba0d-98a869f7d839.png">

**Comment by Howard20181** [comment]
PixEz能用的，需要把不要绕过SNI嗅探打开

**Comment by Howard20181** [comment]
Shaft也能用

**Reply by iPixelOldC** [reply]
woc 那可能就是我这边不知道干了啥把webview给整坏了吧，我哪天重装试试，谢谢啊

**Reply by Howard20181** [reply]
你可以到Play商店装个WebView，谷歌的比较新
